src/tests/JIT/Regression/JitBlue/Runtime_105474/Runtime_105474.cs

```diff
@@ -0,0 +1,71 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Runtime.CompilerServices;
+using System.Numerics;
+using System.Runtime.Intrinsics;
+using System.Runtime.Intrinsics.Arm;
+using Xunit;
+
+public class Program
+{
+    public static Vector<double> s_3;
+
+    [Fact]
+    public static void TestMethod1()
+    {
+        if (Sve.IsSupported)
+        {
+            var vr1 = Vector128.CreateScalar((double)10).AsVector();
+            s_3 = Sve.FusedMultiplyAdd(vr1, s_3, s_3);
+        }
+    }
+
+    [Fact]
+    public static void MoreTestMethods()
+    {
+        if (Sve.IsSupported)
+        {
+            TestMethod2(Vector<double>.Zero);
+            TestMethod3(Vector<double>.Zero);
+            TestMethod4(Vector<double>.Zero);
+            TestMethod5(Vector<double>.Zero);
+            TestMethod6(Vector<double>.Zero);
+        }
+    }
+
+    [method: MethodImpl(MethodImplOptions.NoInlining)]
+    private static void TestMethod2(Vector<double> mask)
+    {
+        var vr1 = Vector128.CreateScalar((double)10).AsVector();
+        s_3 = Sve.ConditionalSelect(mask, Sve.FusedMultiplyAdd(vr1, s_3, s_3), s_3);
```

Comment:
Is this testing what we want?

`s_3` is a static field, so I'd expect this is relying extra on CSE and other opts to get the "right shape"

`Sve.FusedMultiplyAdd` should also be capable of emitting either `FMAD` (`Zdn = Za + Zdn * Zm`) or `FMLA` (`Zda = Zda + Zn * Zm`), which is to say between these two instructions any operand can be the RMW one, so it might be good to ensure we're covering the range here with some comments or even assertions of what codegen is expected

Reply:
I hit an assert with this scenario so added it.

Reply:
@tannergooding how would we go about explicitly validating the codegen for this? On my machine, the decision to use fmad or fmla for a test changes depending on if we're optimizing or not (though I'm not hitting asserts either way with this change).

Reply:
By utilizing https://github.com/dotnet/runtime/blob/main/docs/workflow/testing/coreclr/disasm-checks.md

We should be able to add tests that validate each of the operands given `Sve.FusedMultiplyAdd(a, b, c)` end up as the target. We should then be able to further expand those base tests over `ConditionalSelect` to ensure the right behavior still occurs.

It might be a release and TieredCompilation=0 only test, but it should be possible to get setup and validated.

Reply:
@tannergooding thanks for pointing that out -- I've updated the tests to check for expected codegen.